src/test/CharterManager-unit.t.sol

```diff
@@ -0,0 +1,644 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.12;
+
+import "./TestBase.sol";
+import "src/CharterManager.sol";
+
+import {Vat} from "dss/vat.sol";
+import {Jug} from 'dss/jug.sol';
+import {Vow} from 'dss/vow.sol';
+import {Spotter} from "dss/spot.sol";
+import {DaiJoin} from 'dss/join.sol';
+import {DSValue} from 'ds-value/value.sol';
+import {ManagedGemJoin} from "dss-gem-joins/join-managed.sol";
+
+interface VatHelpersLike {
+    function urns(bytes32, address) external view returns (uint256, uint256);
+    function dai(address) external view returns (uint256);
+    function gem(bytes32, address) external view returns (uint256);
+}
```

Comment:
Since you're importing the Vat directly you can eliminate this and cast to `Vat` when needed.

Reply:
Done